timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -231,6 +231,16 @@ void declare_context(py::module &m) {
             [](timemachine::Context &ctxt, const py::array_t<double, py::array::c_style> new_x_t) {
                 ctxt.set_x_t(new_x_t.data());
             })
+        .def(
+            "set_v_t",
```

Comment:
For these two new wrapped functions should we verify that the velocities and box are of the correct shape? The box seems easiest, though making sure we don't corrupt the length of velocities seems handy.

This probably goes for all of our setters... though not necessary for this PR

Reply:
Good call. Runtime errors caused by passing in position, box, or parameter arrays of the wrong size can be catastrophic and should be prevented as early as possible.

I should fix in this PR, since the PR multiplies the opportunities to encounter this sort of problem!

(For example, if I pass in a 3 x 2 `box`, no error will be raised by the new setter, but the context will have a 3 x 3 `box` with zeros for the missing entries. If I pass in a 3 x 4 `box`, no error will be raised by the new setter but the context will have a 3 x 4 `box`...)